lib/components/primitive-components/Trace.ts

```diff
@@ -623,18 +610,26 @@ export class Trace extends PrimitiveComponent<typeof traceProps> {
       }
     }
 
-    for (const { port } of ports) {
-      connection.pointsToConnect.push({
-        ...port._getGlobalSchematicPositionAfterLayout(),
-        ...projectPointInDirection(
-          port._getGlobalSchematicPositionAfterLayout(),
-          port.facingDirection!,
-          0.1501,
-        ),
-        layer: "top",
-      })
+    // Get port positions for later use
+    const portData = ports.map(({ port }) => ({
```

Comment:
PortsWithPosition


(Im anti-“data”, its basically the same as portsVariable)